collector/mdadm_linux.go

```diff
@@ -152,6 +152,8 @@ func parseMdstat(mdStatusFilePath string) ([]mdStatus, error) {
 	mdStates := make([]mdStatus, 0, estimateMDs)
 
 	for i, l := range lines {
+		active, total, size = 0, 0, 0
```

Comment:
Ahh, crud, you're right.  They're currently defined as int64.  I guess for now 0 is a reasonable solution.